docs/package_templates/prebuilt_tool_package/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+from conan import ConanFile
+from conan.tools.files import get, copy
+import os
+
+
+required_conan_version = ">=1.47.0"
+
+
+class PackageConan(ConanFile):
+    name = "package"
+    description = "short description"
+    license = "" # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/project/package"
+    topics = ("topic1", "topic2", "topic3", "pre-built") # no "conan"  and project name in topics
+    settings = "os", "arch", "compiler", "build_type" # even for pre-built executables
+
+    # not needed but supress warning message from conan commands
+    def layout(self):
+        pass
+
+    # specific compiler and build type, usually are not distributed by vendors
+    def package_id(self):
+        del self.info.settings.compiler
+        del self.info.settings.build_type
+
+    # in case some configuration is not supported
+    def validate(self):
+        if self.info.settings.os == "Windows":
+            self.info.warn(f"{self.ref} is not well tested on Windows.")
```

Comment:
why not raise `ConanInvalidConfiguration`?

Reply:
I didn't add ConanInvalidConfiguration, because we offer Windows support in conandata.yml. 

I wanted to use validate(), but not blocking an OS, so I remembered about Gtest in the past, which we had a warning about not well tested configuration.

It can not be an option, because, we don't use options here. So, it should be a setting.